alf/examples/ppo_icubwalk.gin

```diff
@@ -29,6 +29,7 @@ create_ac_algorithm.value_fc_layers=(256, 128)
 create_ac_algorithm.learning_rate=1e-4
 
 # training config
+TrainerConfig.unroll_length=20
 TrainerConfig.mini_batch_length=1
 TrainerConfig.num_steps_per_iter=16384
```

Comment:
fixed .   the original gin file did not config `unroll_length` and use default value  defined in `off_policy_trainer.trian`

Reply:
num_steps_per_iter should removed because it's not used for sync_off_policy_trainer. num_steps_per_iter was used by an old version of OffPolicyTrainer, that's the reason it is here.

Reply:
removed